src/frontend/Sider/Behandling/Venstremeny/Oppsummering/StønadstypeTag.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { Tag } from '@navikt/ds-react';
+
+import { Behandling } from '../../../../typer/behandling/behandling';
+import { formaterEnumVerdi } from '../../../../utils/tekstformatering';
+
+export function StonadstypeTag({ behandling }: { behandling: Behandling }) {
+    return (
+        <div>
+            <Tag size="small" variant="neutral">
+                {formaterEnumVerdi(behandling.stønadstype)}
+            </Tag>
+        </div>
```

Comment:
Tror ikke denne div'en trengs? 
```suggestion
            <Tag size="small" variant="neutral">
                {formaterEnumVerdi(behandling.stønadstype)}
            </Tag>
```